plugins/node/opentelemetry-instrumentation-restify/package.json

```diff
@@ -56,7 +56,7 @@
     "@types/semver": "^7.3.12",
     "mocha": "7.2.0",
     "nyc": "15.1.0",
-    "restify": "8.6.1",
+    "restify": "^11.1.0",
```

Comment:
I think renovate will create a followup PR to pin this dependency. Might be better to bump it to new version but keep it pinned